numpy/char/__init__.py

```diff
@@ -1,2 +1,5 @@
 from numpy._core.defchararray import __all__, __doc__
 from numpy._core.defchararray import *
+# import slice function directly from strings module instead
+# of adding it to the deprecated defchararray module
+from numpy._core.strings import slice
```

Comment:
Oh oops missed this. On second thought I think we should just delete it. If you really do feel strongly that having the function be available in the `np.char` namespace is important, you should add it to `__all__` too.

Reply:
It's not really that `defchararray.py` is deprecated, we want to get rid of the entire `np.char` namespace since it's also associated with the CharArray concept which we want to move away from.